Gemfile.lock

```diff
@@ -23,6 +23,7 @@ GEM
     azure_mgmt_resources (0.17.2)
```

Comment:
We shouldn't commit the Gemfile.lock.

Reply:
Let's add this to `.gitignore`. Our gemspec file is responsible for spelling out the correct dependencies.